app/Http/Controllers/LoadoutsController.php

```diff
@@ -14,21 +15,32 @@ class LoadoutsController extends Controller
 {
     public function index(Request $request)
     {
+        // dd($request);
         SEOTools::setTitle('Deep Rock Galactic Loadouts');
-
         $loadouts = Loadout::sortable(['updated_at' => 'desc'])
             ->filter($request->all())
             ->with('mods', 'mods.gun', 'character', 'creator')
             ->withCount('votes')
             ->paginate();
         $characters = Character::orderBy('name')->get();
+        $overclocks = Overclock::orderBy('overclock_name')->get();
+        $primaries = Gun::where('character_slot', 1)->orderBy('name')->get();
+        $secondaries = Gun::where('character_slot', 2)->orderBy('name')->get();
```

Comment:
For these we can probably use `pluck()` to lighten the payload a little bit. That would just create a key value pair for each entry.

Reply:
Dope